flexget/utils/imdb.py

```diff
@@ -239,10 +239,10 @@ def parse(self, imdb_id):
             return
 
         # Parse stuff from the title-overview section
-        name_elem = title_overview.find('h1', itemprop='name')
-        self.name = name_elem.find(text=True, recursive=False).strip()
+        name_elem = title_overview.find(itemprop='name')
+        self.name = name_elem.contents[0]
 
-        year = name_elem.find('a')
+        year = title_overview.find(attrs={'class': 'nobr'}).find('a', href=True)
```

Comment:
Is the nobr class always there? This will crash otherwise. Maybe change to the selector format so each level doesn't have to be checked for existence separately?